lib/testing/src/lib/test.configuration.ts

```diff
@@ -31,6 +31,11 @@ export interface TestConfiguration {
         password: string
     };
 
+    identitySalesUser: {
```

Comment:
I don't think we need an identity as a prefix but I don't see we use it anywhere in ADF why is it needed?